internal/providers/terraform/aws/ebs_volume.go

```diff
@@ -38,9 +38,25 @@ func NewEBSVolume(d *schema.ResourceData, u *schema.ResourceData) *schema.Resour
 }
 
 func ebsVolumeCostComponents(region string, volumeApiName string, gbVal decimal.Decimal, iopsVal decimal.Decimal) []*schema.CostComponent {
+	var name string
```

Comment:
I think I prefer having brackets at the end, despite it not matching AWS' names, so we keep consistency in infracost, see https://docs.google.com/spreadsheets/d/1H_bn2jLzYr7xyrvNsFn-0rDaGPGpnrVTPsjVHzr-kM4/edit#gid=0 Happy to change the gdoc if you prefer brackets in the middle